deploy-board/deploy_board/webapp/cluster_view.py

```diff
@@ -320,6 +320,11 @@ def post(self, request, name, stage):
             return HttpResponse(e, status=500, content_type="application/json")
         return HttpResponse(json.dumps(settings), content_type="application/json")
 
+def promote_image(request, image_id):
+    baseimages_helper.promote_image(request, image_id)
```

Comment:
`return HttpResponse()` to avoid warnings.

Reply:
Perfect, thanks